src/lib.rs

```diff
@@ -83,6 +89,127 @@ where
     }
 }
 
+// those functions are provided by the afl-llvm-rt static library
+extern "C" {
+    fn __afl_persistent_loop(counter: usize) -> isize;
+    fn __afl_manual_init();
+}
+
+/// Fuzz a closure by passing it a `&[u8]`
+///
+/// This slice contains a "random" quantity of "random" data.
+///
+/// ```rust,ignore
+/// # extern crate afl;
+/// # use afl::fuzz;
+/// # fn main() {
```

Comment:
I'm fine as-is, but considering users are going to need the `main` function, I wouldn't be opposed to showing it for these examples.